data/presets/building.json

```diff
@@ -19,6 +19,8 @@
         "not/name",
         "operator",
         "roof/colour",
+        "roof/shape",
+        "roof/height",
```

Comment:
If you want to be fancy about it, you could extract those three fields into a template to share them, see https://github.com/openstreetmap/id-tagging-schema/tree/main/data/presets/%40templates

That will make it easier to not forget one later…